projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/group-skipper/group-skipper.config.ts

```diff
@@ -0,0 +1,43 @@
+import { testProductUrl } from '../tabbing-order';
+
+export interface GroupSkipperPageConfig {
```

Comment:
I would rename these props to:
`pageUrl` and `elementsCount`/`elementsNumber`/`?` (or something similar)? url and length are imho too generic